packages/terra-embedded-component/src/EmbeddedComponent.jsx

```diff
@@ -1,31 +1,144 @@
+/* global SystemJS */
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
-import styles from './EmbeddedComponent.scss';
+import 'systemjs/dist/system';
+import { loadStylesheet } from './assets';
 
-const cx = classNames.bind(styles);
+// Expose AMD required globals on the window
+window.define = window.SystemJS.amdDefine;
+window.require = window.requirejs = SystemJS.amdRequire;
+
+const DefaultTimeout = () => <p>Component mounting is timed out.</p>;
 
 const propTypes = {
- /*
- * Content to be displayed as the name
- */
-  name: PropTypes.string,
+
+ /**
+  * The basePath to load the manifest and manifest asset dependencies from.
+  * */
+  basePath: PropTypes.string.isRequired,
+
+  /**
+   * A placeholder to display while the component is laoding.
+   * */
+  placeholder: PropTypes.element,
+
+  /**
+   * A component to display while the loading of css or module is timed out.
```

Comment:
I think this makes it a bit better

` A component to display when the css or module times out`